draft-ietf-tls-rfc8446bis.md

```diff
@@ -3517,6 +3517,16 @@ a KeyUpdate with the old key is received before accepting any messages
 encrypted with the new key. Failure to do so may allow message truncation
 attacks.
 
+With a 128-bit key as in AES-128, rekeying 2^64 times has a high
```

Comment:
```suggestion
With a 128-bit key as in AES-128, rekeying 2<sup>64</sup> times has a high
```

You might like to do this throughout.  It does read better (in HTML) and the text output remains the same.